practice/src/javascripts/helpers/api.js

```diff
@@ -0,0 +1,22 @@
+import { API } from '../constants/apiUrl';
+
+const APIHelper = {
+  /*
+   * Fetches data from the API using the specified endpoint.
+   * It defines an asynchronous function 'get' to fetch data from the API.
+   */
```

Comment:
You should install the extension `Better Comments`  in VSCode and update format comment here base on this extension